### ISSUE TITLE
Implement Search Functionality for Sticky Notes

### ISSUE THREAD

**Issue by narainkarthikv** [narainkarthikv/Sticky-Memo#2]
We want to add a search functionality to the application that allows users to quickly find their sticky notes by title or content. This issue will help beginners learn about state management and handling user input in React. Currently Only notes has this feature - implement tables, boards

Task:

1. Add a search bar to the application.
2. Implement a search function that filters sticky notes based on the input in the search bar.
3. Display the filtered sticky notes in real-time as the user types in the search bar.

**Comment by m1her** [comment]
but the website already has a filter, could you provide more details please, i'm interested

**Comment by narainkarthikv** [comment]
Hi, the search filter is already provided only for the sticky notes component and is not implemented for tables and board components. 

If possible ping me with new UI ideas if you have any, so the UI/UX can be smoother.

**Comment by m1her** [comment]
Hey. I think there is so much ideas for the UI you can apply, just have a look over google and search for some designs. 

You, could justify the nav's bar padding and font weight.
And i think there is so much rounded stuff and there is no obvious separations with borders or bg colors.
You could make the sticky notes look like a sticky note with random bg colors.

assign me to any of the issues and i will be happy to contribute and help,

Thank You.

**Comment by narainkarthikv** [comment]
Yeah sure, Can you help me with the board component issue and with more UI enhancement?

**Comment by m1her** [comment]
Of course.
